content/full-day/6-code.md

```diff
@@ -78,18 +78,22 @@ Adding the filters to the page will require updating a minimum of three files -
    - **server/app.py**
    - **server/test_app.py**
    - **client/src/components/DogList.svelte** 
-2. Select **Copilot Edits** in the Copilot Chat window.
-3. Ensure **Claude 3.7 Sonnet** is selected for the model.
-4. Select **Add Context...** in the chat window and **Open Editors** from the prompt. This will add all currently open files to the context.
-5. Ask Copilot to perform the operation you want, to update the page to add the filters. It should meet the following requirements:
+2. Open GitHub Copilot Chat.
+3. Switch to edit mode by selecting **Edit** in the chat mode dropdown at the bottom of Chat view (should be currently **Ask**)
+4. Ensure **Claude 3.7 Sonnet** is selected for the model.
+5. Select **Add Context...** in the chat window.
+6. Select **server/app.py**, **client/src/components/DogList.svelte** and **server/test_app.py** files (you need to select **Add context** for each file) 
```

Comment:
Curious the rationale behind this update. I personally find it easier to go with the "add all open editors" option

Reply:
I would too, but it doesn't exist anymore. :) 

Unless I'm missing something very obvious

Reply:
Weird. I just checked, and it's on the stable but not insiders. Let's roll with what you've got here